enterprise/frontend/src/metabase-enterprise/data-studio/common/components/Table/Table.module.css

```diff
@@ -0,0 +1,56 @@
+.Row {
```

Comment:
In other css modules e.g. `enterprise/frontend/src/metabase-enterprise/data-studio/app/pages/DataStudioLayout/DataStudioLayout.module.css` lowercase names are used but capitalized here